test/services/ProgressServiceSpec.scala

```diff
@@ -29,15 +29,28 @@ import org.mockito.Mockito._
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.mock.MockitoSugar
 import org.scalatestplus.play.{OneAppPerSuite, PlaySpec}
+import play.api.inject._
+import play.api.{Application, Mode}
+import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.mvc.Call
 import play.api.test.Helpers._
 import uk.gov.hmrc.play.frontend.auth.AuthContext
 import uk.gov.hmrc.play.http.HeaderCarrier
+import utils.GenericTestHelper
 
 import scala.concurrent.ExecutionContext.Implicits._
 import scala.concurrent.{ExecutionContext, Future}
 
-class ProgressServiceSpec extends PlaySpec with MockitoSugar with ScalaFutures with OneAppPerSuite {
+class ProgressServiceSpec extends GenericTestHelper with MockitoSugar with ScalaFutures with OneAppPerSuite {
+
+  val paymentsConnector = mock[PayApiConnector]
```

Comment:
Would it be possible to take this out? ProgressService won't use the payments connector and this could confuse things with it being in the test.